### PR TITLE
fix : Fixed UI changes [123]

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description

Fixed bugs from the excel sheet

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@library/icons/VerticalGrid.svelte [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+    export let height: number = 24;
+    export let width: number = 24;
+    export let color: string = "#45494D";
+  </script>
```
Comment (itsmdasifraza) [inline src/packages/@library/icons/VerticalGrid.svelte:5]: use default color as grey

File: src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte [hunk 2]
```diff
@@ -154,7 +154,7 @@
     class="ms-2 save-disk d-flex align-items-center justify-content-center border-radius-2 border-0"
     on:click={handleSaveRequest}
   >
-    <DiskIcon height={20} width={20} />
+    <DiskIcon height={22} width={22} />
   </button>
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte:158]: add color props

File: src/packages/@workspaces/features/tab-bar/layout/TabBar.svelte [hunk 3]
```diff
@@ -189,7 +191,11 @@
             viewChange = !viewChange;
           }}
         >
-          <ViewGrid height={15} />
+          {#if $requestSplitterDirection === "horizontal"}
+            <ViewGrid color={"var(--text-primary-400)"} height={15} />
+          {:else}
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/tab-bar/layout/TabBar.svelte:196]: use width prop

Comment (isha-dubey) [reply src/packages/@workspaces/features/tab-bar/layout/TabBar.svelte:196]: Resolved

File: src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte (hunk 2 above)
Comment (isha-dubey) [reply src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte:158]: Resolved.

**Comment by punitkashyup** [comment]
Please attach the issue link

**Pull request merged**